### DISCUSSION TITLE
![Help](how_to_add_inline_image.png) ???

### DISCUSSION THREAD

**Discussion by akashpal-21** [xwmx/nb#270] [Q&A] [answered]
I followed the instruction on the landing page that to add an image i need to import; i imported image1.png into home
when I want to add the image in a text I typed `![](image1.png)` but in browse mode, in gui mode, or anywhere, the image is not rendered.. What am I doing wrong???

**Comment by xwmx** [comment]
@akashpal-21 I appreciate your creative title! 😉 I just tried it and referencing images [within the same notebook by relative path](https://github.com/xwmx/nb#inline-images) works as expected and displays the image on the item page in `nb browse --gui`. For example, if I import `example.png` so it's in the root level of the notebook, I can reference it with a note containing the following and it displays as expected:

```markdown
![](example.png)
```

I'm not sure what would cause it to not show up in `nb browse --gui`. Is the image in the same notebook as the markdown document?

**Reply by akashpal-21** [reply]
@xwmx Yes, the image is in the same notebook. I am adding a video to show the procedure I followed. Please help.
I am using Debian 12 for reference 

https://github.com/xwmx/nb/assets/46517170/a5e04bc9-19bc-45b4-8aeb-35324ac27614

**Comment by akashpal-21** [comment, marked as answer]
IT WORKS AFTER DOING env install
After doing complete install of optionals the image render works !!!!!!
IDK what optional was 'mandatory' for image render, but it works now

Thank you so muchhh. I love this script

**Comment by xwmx** [comment]
@akashpal-21 Awesome. Happy you solved it. Thanks for the video. It looks like it needed `pandoc` to be installed. Glad you like `nb`! 😊
